packages/core/lib/modules/accounts.js

```diff
@@ -188,23 +188,23 @@ export class Accounts {
     assert(!accountExists, Errors.accountAlreadyExists);
 
     if (json) {
-      const pair = await KeyringRpc.addFromJson(json, password);
+      const pair = await keyringService.addFromJson(json, password);
 
       assert(pair && pair.type, 'invalid keypair');
 
-      keyPairType = pair.type;
+      type = pair.type;
```

Comment:
why the rename? Isn't keypairtype more descriptive?

Reply:
type is used by polkadot js library, trying to use the same parameter name

Reply:
Okay, that makes sense.